Formula/mypy.rb

```diff
@@ -2,8 +2,8 @@ class Mypy < Formula
   desc "Experimental optional static type checker for Python"
   homepage "http://www.mypy-lang.org/"
   url "https://github.com/JukkaL/mypy.git",
-    :tag => "v0.4.2",
-    :revision => "b8a2df91707bc3d5c9c3677c8017b614e41d017a"
+      :tag => "v0.4.3",
```

Comment:
I disagree unless our "custom" requirements are being enforced programatically. It would be better to just go with whatever RuboCop natively supports (unless someone is willing to do the work to customize it) rather than chasing people around to fix this or that indentation in every review where they've stepped outside our nebulous lines.